internal/runners/deploy/uninstall/uninstall.go

```diff
@@ -61,6 +62,15 @@ func (u *Uninstall) Run(params *Params) error {
 				"Cannot determine current working directory. Please supply `--path` argument")
 		}
 		path = cwd
+		if runtime.GOOS == "windows" {
+			err = os.Chdir(filepath.Dir(path)) // cannot os.RemoveAll() in current working directory, so leave it
+			if err != nil {
+				return locale.WrapInputError(
+					err,
+					"err_deploy_uninstall_cannot_chdir",
+					"Cannot switch away from current working directory. Please supply `--path` argument")
+			}
+		}
```

Comment:
Should this be an error message rather than us changing the directory for the user? Usually, on Windows, I would expect a "directory in use" error message rather than a program handling it for me.

Reply:
I don't think so, because `state deploy <org/namespace>` without any additional arguments deploys to the current working directory. I think it would be nice to allow for the reverse, hence the need to do this on Windows.